### ISSUE TITLE
plus7, Ninsemsn and Tenplay

### ISSUE THREAD

**Issue by cvet52** [xbmc-catchuptv-au/repo#42]
Australian commercial catch up TV fail to play any content, ABC and SBS are okay

**Comment by andybotting** [comment]
How about some details, like OS, Kodi version and provide logs?

**Comment by cvet52** [comment]
ATV2 IOS 6.1.4  Kodi helix 14.1 Also tried 14.2 both not working

[kodi.zip](https://github.com/xbmc-catchuptv-au/repo/files/448707/kodi.zip)

**Comment by andybotting** [comment]
The issue is:
12:25:51 T:804499456   DEBUG: ffmpeg[2FF3B000]: https protocol not found, recompile with openssl or gnutls enabled.

This version of Kodi on ATV (and Android too I believe) don't support HTTPS, so you're out of luck. I'm not sure this is any better on newer versions for the ATV.

I do remember looking at this issue for Android specifically some time ago and it was hard to fix, which is obviously why the Kodi devs never did it.

**Comment by cvet52** [comment]
Thanks. I appreciate your help

**Comment by glennguy** [comment]
@cvet52 - the plus 7 add-on has an alternate stream handling method in the settings which is designed to work without HTTPS, give that a try.

**Comment by cvet52** [comment]
Thanks for that.
Tried both Native and compatibility mode. Still no playback  Interesting how native mode is labelled as being compatible with Kodi v13 & v14+. I have v14,1 and it is not working should I try v 14.0?

**Comment by glennguy** [comment]
It's compatible if you're not on Android, which doesn't help much I know!

Have you got a log for it failing in compatibility mode? Also (I've
accidentally messed this up a few times) make sure to click the ok button
after changing the settings rather than the back button.

**Comment by cvet52** [comment]
Thanks again  Here is the requested log I am not on Android  Using Apple TV 2.

**Comment by glennguy** [comment]
Ah sorry I misread, but yeah android and ios are in the same boat in this situation.

No log attached?

**Comment by cvet52** [comment]
here is the requested log
[kodi.log.zip](https://github.com/xbmc-catchuptv-au/repo/files/463038/kodi.log.zip)
Does the comment about android apply to later versions of kodi e.g.. v16

**Comment by glennguy** [comment]
No, from what I've seen while quickly chasing this up HTTPS support was
added in Kodi 15.

Looks like it's not going to work with your current setup. Even though the
compatibility mode manages to get around getting the playlist file from a
HTTPS location, it's still embedded with video chunks that all come from
HTTPS locations as well.

Luckily for you, [this](http://chewitt.openelec.tv/appletv/) is a potential
solution, if you're happy to get rid of iOS.

**Comment by andybotting** [comment]
Unfortunately not - that OpenELEC is for the AppleTV1 which is x86 based. The ATV2 is ARM based.

I'd recommend a Raspberry Pi if you want low cost Kodi :+1:

**Comment by glennguy** [comment]
My bad 😳 

Big plus 1 here for Raspberry Pi 2 or 3. Big bonus too if your TV has a good CEC implementation.

**Comment by cvet52** [comment]
Thanks for your efforts. I'm learning quickly.
